models/aeif_cond_alpha.h

```diff
@@ -103,6 +103,11 @@ and
 For the reference implementation of this model, see
 `aeif_models_implementation <../model_details/aeif_models_implementation.ipynb>`_ notebook.
 
+.. note::
+
+   A maximum integration step size of 10 μs is enforced, regardless of the
+   resolution set in the NEST kernel.
```

Comment:
This is not correct. `init_buffers_()` initialises `B_.IntegrationStep_` to `min(0.01, Time::get_resolution().get_ms())`, but after that the adaptive stepsize mechanism controls `IntegrationStep_`. The initialization line has been unchanged since 2.6.1 at least, and I am not sure why we have it at all, given that we are using an adaptive stepsize solver. It might be a leftover from very early versions with a fixed-stepsize solver.